### ISSUE TITLE
General Treatment of `h*` Headings

### ISSUE THREAD

**Issue by dmolsen** [wvuweb/undergraduate-admissions#33]
I'm not sure if this is a brand treatment issue or not...

On `h2`s it'd be nice if there was more padding above the `h2` and a little tighter in padding below. The visual hierarchy feels out of whack on something like an "area of interest":http://admissions.sandbox.wvu.edu/academics/areas-of-interest/art-and-design-program or "major":http://admissions.sandbox.wvu.edu/academics/majors/art-education. The various audience pages feel the same way.

**Comment by dustinmazon** [comment]
I agree, we should look at tightening up the typography site-wide. I am going to compare the mockup I did to the typographic styles on the site. I think the spacing I had going on in my mockup were better. Here is a test of the full typographic settings for the site. 

http://admissions.sandbox.wvu.edu/home/typography-test

Here is what I am thinking about doing: 

- [ ] Use Helvetica Bold 75 for `<p class="lede"><strong>` or  `<p class="lede"><b>`.
- [ ] Add a little more line-height to the `<p class="lede">`.
- [ ] Make sure line-height is good for all Heading tags when escaped to two lines of text.
- [ ] Test scalability of text responsiveness. Looks like some headings and styles are too big for mobile.
- [ ] Styles for Callout Panels: see contextual backgrounds section http://getbootstrap.com/css/#helper-classes-backgrounds will need these for highlighting important information related to "How to Apply" section.
- [ ] Introduce some color in the headings.
- [ ] More margin top and line-height for some of the headings.

I'm sure there will be more things to address once I get into it, like when the heading styles are used on a sidebar.

## Example

![screen shot 2015-07-10 at 11 09 28 am](https://cloud.githubusercontent.com/assets/1126703/8621549/29bd85aa-26f4-11e5-8574-6fce3e520384.png)

**Comment by dmolsen** [comment]
Since this is related to type...  there's a width (1300px or so) where the `.shout` text comes up hard left on the edge.
